README.md

```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
+furyctl create cluster --phase infrastructure
 ```
 
-### Installers
+If you choose to create a VPN in the infrastructure phase, you can automatically connect to it by using the flag `--vpn-auto-connect`.
 
-To deploy all the infrastructure components `furyctl` uses *installers*.
+```bash
+furyctl create cluster --phase kubernetes
+```
 
-> You can use an environment variable to avoid passing the token via console: `FURYCTL_TOKEN`.
+After running the command, remember to export the `KUBECONFIG` environment variable to point to the generated kubeconfig file or
+to use the flag `--kubeconfig` in the following command.
 
-Contact [sales@sighup.io](mailto:sales@sighup.io) to get more details about this feature.
+```bash
+furyctl create cluster --phase distribution
+```
 
-#### Bootstrap
+##### Infrastructure
 
-The available `bootstrap` provisioners are:
+The available `infrastructure` provisioners are:
 
-| Provisioner | Description                                                                                                                                            |
-| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `aws`       | It creates a VPC with all the requirements to deploy a Kubernetes Cluster. It also includes a VPN instance easily manageable by using `furyagent`.     |
+| Provisioner | Description                                                                                                                                        |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `aws`       | It creates a VPC with all the requirements to deploy a Kubernetes Cluster. It also includes a VPN instance easily manageable by using `furyagent`. |
 
-#### Clusters
+##### Kubernetes
 
-The available `cluster` provisioners are:
+The available `kubernetes` provisioners are:
 
-| Provisioner | Description                                                                                                                              |
-| ----------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| `eks`       | Creates an EKS cluster on an already existing VPC. It uses the [fury-eks-installer](https://github.com/sighupio/fury-eks-installer)      |
+| Provisioner | Description                                                                                                                         |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `eks`       | Creates an EKS cluster on an already existing VPC. It uses the [fury-eks-installer](https://github.com/sighupio/fury-eks-installer) |
```

Comment:
this seems to be related to the old furyctl

Reply:
removed